fetch_bringup/launch/include/teleop.launch.xml

```diff
@@ -2,6 +2,7 @@
 
   <arg name="joy_device" default="/dev/fetch_joy"/>
   <arg name="ps4" default="false"/>
+  <arg name="is_fetch" default="false"/>
```

Comment:
Doesn't look like this is used?

Reply:
Ahh, missed that in cleanup. Thanks!

At one point I had a bunch of added logic, before realizing why things weren't working at one point.  Why things weren't working: I ran freight.launch which sets /teleop/is_fetch to false, and then ran fetch.launch again later, and I could only teleop the base.... And that's why I now have the is_fetch = True line later in this launch file.